web/src/components/App/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Route, Switch } from 'react-router-dom'
+import { ThemeProvider } from 'emotion-theming'
+import { theme } from 'theme'
+import Navbar from 'components/Navbar'
+
+import routes from 'routes'
+
+import 'semantic-ui-css/semantic.min.css'
```

Comment:
Importing this module propagates two changes for jest.
1. Create a mock style file as in [/__ mocks __/styleMock.js](https://github.com/openoakland/woeip/blob/7abc804a8bfbc6821a9c355698aeb552c3a6fc44/web/__mocks__/styleMock.js#L1)
2. reference said file in package.json moduleNameMapper
   - [`"^.+\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"`](https://github.com/openoakland/woeip/blob/7abc804a8bfbc6821a9c355698aeb552c3a6fc44/web/package.json#L69)
    - P.S: great call on keeping moduleNameMapper